Hardware/Hardware/CardReader/StripeCardReader/RefundParameters+Stripe.swift

```diff
@@ -0,0 +1,24 @@
+#if !targetEnvironment(macCatalyst)
+import StripeTerminal
+
+extension Hardware.RefundParameters {
+    /// Initializes a StripeTerminal.PaymentIntentParameters from a
```

Comment:
Should it read `StripeTerminal.RefundParameters`?

Reply:
The same mismatch seems to happen in most comments in this extension

Reply:
Oops, good spot. Thanks!